### DISCUSSION TITLE
Verwendete Modelle

### DISCUSSION THREAD

**Discussion by Trunks1982** [Trunks1982/Daikin-Rotex-HPSU-CAN#50] [Feedback]
Hallo,
Ich hätte mal eine Bitte an euch.
Mich würde es mal interessieren was für Versionen von Modellen ihr verwendet.
So könnte ich mal eine Kompatibilitätsliste erstellen.
Bitte komplette Modell Bezeichnung.

**Comment by Trunks1982** [comment]
Rotex HPSU Compact 508 8kw Baujahr 2017

**Comment by Tom88s** [comment]
Rotex HPSU Compact 516 11kW

**Comment by MichaelGer1410** [comment]
Rotex HPSU Compact 508 8 KW 
Baujahr 2013

**Comment by tosomax** [comment]
Daikin Altherma - EHSXB-B
ehsxb08p50ba
Model 2018

**Comment by svde** [comment]
Rotex HPSU Compact 508 BIV 6 KW (2016)
M5stack [AtomS3U ESP32S3 Development Kit with USB-A](https://shop.m5stack.com/products/atoms3u)
M5stack [Mini CAN Unit (TJA1051T/3)](https://shop.m5stack.com/products/mini-can-unit-tja1051t-3)

**Comment by ap20132022** [comment]
Daikin ALTHERMA 3R ECH2O  (Rotex HPSU Compact 308) 4KW (2020)
